cmake/Format.cmake

```diff
@@ -56,7 +56,7 @@ function(gw_enable_formatting_targets)
         set(format_check_list "")
 
         foreach(source IN LISTS FOLDER_FORMAT_SOURCES)
-            file(RELATIVE_PATH RELATIVE_PATH ${CMAKE_SOURCE_DIR} ${source})
+            file(RELATIVE_PATH RELATIVE_PATH ${PROJECT_SOURCE_DIR} ${source})
```

Comment:
Since we have only one project now, maybe we should also change `CMAKE_CURRENT_SOURCE_DIR`  to `PROJECT_SOURCE_DIR`  in  https://github.com/clara-parabricks/GenomeWorks/blob/74e6424c156a7ee15b4137d4788a4257ee6482c4/CMakeLists.txt#L91

Reply:
`CMAKE_CURRENT_SOURCE_DIR` tracks the current directory in which the CMakeLists.txt file is being processed. `PROJECT_` doesn't have an equivalent to it since it's just tracking the current dir. `PROJECT_SOURCE_DIR` is the replacement for `CMAKE_SOURCE_DIR` though, since it'll track the root folder where `project` was called instead of where the top level `cmake` command was called from.